public/views/partials/footer.html

```diff
@@ -0,0 +1,17 @@
+<footer id="footer">
+	<div class="scroll-to-top">
+		<a href="" onclick="scrollToTop();">Top</a>
+	</div>
+	<div class="footer-container">
+		<ul class="footer-link-collection">
+            <li><a class="footer-link" href="/contact">Contact Us</a></li>
+            <li><a class="footer-link" href="/media">Social Media</a></li>
+            <li><a class="footer-link" href="/sponsors">Sponsors</a></li>
```

Comment:
Instead of a social media button, could you but a direct link to a facebook page? We currently don't have one, so don't worry about that, but I think it would be better to have a FB glyphicon instead of a text link. To do that you would need to import Font Awesome, because bootstrap doesn't have a facebook glyphicon. Look at the first answer here: http://stackoverflow.com/questions/25767708/glyphicon-for-facebook-not-showing-up